client/src/elements/item-preview/item-preview.js

```diff
@@ -13,20 +13,9 @@ export class ItemPreview {
   @bindable id;
   @bindable target;
 
-  sizeOptions = [
```

Comment:
keep the default and let it be overwritten in the config. otherwise this is a breaking change.

Reply:
Implemented in https://github.com/nzzdev/Q-editor/pull/119/commits/c06fdb3a27c77b02acb0fa8af192f8f8ea9ef691